core/pom.xml

```diff
@@ -81,6 +81,18 @@
         </configuration>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
FWI, I've seen people recommending/advocating specifying it anyway because configuration in a `settings.xml` (default groups) could end up resolving to a different plugin.